docs/tutorial_bcc_python_developer.md

```diff
@@ -67,14 +67,14 @@ from bcc import BPF
 # define BPF program
 prog = """
 int hello(void *ctx) {
-	bpf_trace_printk("Hello, World!\\n");
-	return 0;
+    bpf_trace_printk("Hello, World!\\n");
+    return 0;
```

Comment:
Any reason you changed all indentations?

Reply:
The patch copied over what the example codes currently have and that indentation is what the examples currently have.  However, commit 44ad8a1ececed09e41dea5f8622759b9f772aa61 for docs/tutorial_bcc_python_developer.md the comment says:

    Always use spaces in C in the tutorial

There are still tabs in tutorial_bcc_python_developer.md.  Should the remaining tabs in the patch be converted to spaces to be consistent?